Makefile

```diff
@@ -320,8 +320,8 @@ endif
 # Test apply ou map cr
 .PHONY: create-ou-map
 create-ou-map:
-	# Create OU map
-	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -f hack/templates/aws_v1alpha1_configmap.tmpl | oc apply -f -
+# Create OU map
+	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -p ACCOUNTLIMIT="1" -f hack/templates/aws_v1alpha1_configmap.tmpl | oc apply -f -
```

Comment:
Why is this 1?

Reply:
I don't know. @rogbas could you clarify? https://issues.redhat.com/browse/OSD-5095

Reply:
AFAIK this number _used to be_ important in the past, but with the current implementation it doesn't matter if it's `0` or `1` anymore. That has been told, I would prefer to go with `0` since it's the typical initialisation value in programming.

Reply:
We should set it to 0. The 1 value was a reminiscence from the previous solution